docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/{{ cookiecutter.module_name }}/migrations/0001_initial.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.11 on 2022-04-21 18:23
+
+from django.db import migrations, models
+import django.db.models.deletion
+import my_proj.blocks
```

Comment:
Looks like you need to use the `module_name` Cookiecutter var here instead of a hard coded placeholder.